Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/IdentityProvidingClients/IdentityProvidingSSOOIDCClient.swift

```diff
@@ -0,0 +1,17 @@
+//
```

Comment:
The protocol for SSO OIDC client, used for dependency inversion. The concrete struct that conforms to this protocol is code generated at `Sources/Core/AWSSDKIdentity/Sources/InternalAWSSSOOIDC/IdentityProvidingSSOOIDCClient.swift`.